linode/manager
# M3-5772: useEditableLabelState hook

## Description
Prompted by https://github.com/linode/manager/pull/8273#discussion_r826042189, this PR adds a `useEditableLabelState` hooks and replaces instances where `editableLabelState.ts` was used previously.

## How to test
Try to edit the label of Databases and Linodes from their breadcrumbs. Ensure that any appropriate errors display, nothing is logged to the console, and that changes go through as intended.
